make_wininsts.bat

```diff
@@ -1,10 +1,10 @@
-SET PATH=C:\Python26;%PATH%
```

Comment:
@klauer I am not concerned about seeing the thrashing, I am concerned that it is happening.